types/repository.ts

```diff
@@ -1,23 +1,27 @@
-import { z } from 'zod';
-import { ZEnrollment } from './enrollment';
-import { ZInstallation } from './installation';
-import { ZPointTransaction } from './pointTransaction';
+import { z } from "zod";
 
-export const ZRepository = z.object({
-  id: z.string(),
-  githubId: z.number().int(),
-  name: z.string(),
-  description: z.union([z.string(), z.null()]).optional(),
-  homepage: z.union([z.string(), z.null()]).optional(), 
-  topics: z.array(z.string()), 
-  default_branch: z.string().optional(), 
-  installationId: z.string(),
-  configured: z.boolean().default(false),
-  levels: z.any(), 
+import { ZEnrollment } from "./enrollment";
+import { ZInstallation } from "./installation";
+import { ZPointTransaction } from "./pointTransaction";
 
-  pointTransactions: z.array(ZPointTransaction),
-  installation: ZInstallation,
-  Enrollment: z.array(ZEnrollment),
-}).strict();
+export const ZRepository = z
+  .object({
+    id: z.string(),
+    githubId: z.number().int(),
+    name: z.string(),
+    description: z.union([z.string(), z.null()]).optional(),
+    homepage: z.union([z.string(), z.null()]).optional(),
+    topics: z.array(z.string()),
+    default_branch: z.string().nullable().optional(),
+    installationId: z.string(),
+    configured: z.boolean().default(false),
+    logoHref: z.union([z.string(), z.null()]).optional(),
+    projectDescription: z.union([z.string(), z.null()]).optional(),
+    levels: z.any(),
```

Comment:
Avoid `any`. Can we provide more information? Are these a list of strings (`z.array(z.string)`)? Or more objects?

Reply:
its a placeholder bc @chigala works on the levels right now